etest/docker/container.py

```diff
@@ -50,6 +50,14 @@ def create(overlay: str, *args, **kwargs):
     return container
 
 
+def create_simple(*args, **kwargs):
+    """Create a simple docker container."""
```

Comment:
I dont quite get what you mean. They do the same thing but with different api endpoints, so this is the cleanest way to implement it I can come up with

Reply:
Just different endpoints?  I thought I read it as a whole different container definition.  Maybe I'm thinking of that wrong.  It still feels weird to have two create methods in the container module.  What about a Container dataclass that you pass with configuration and you can have a default constructor for Container and a simple constructor or something like that.  Does that make more sense?

Reply:
That certainly sounds like create_simple should be the exposed interface to me.  Do we need both interfaces and if so, why wouldn't having an Optional parameter with configuration suffice?  That's the cleanest and most extensible interface we can provide.

Reply:
The most used function is create(), since the tests we are running require deeper bind mount support. We can revisit this on a future issue but for now I think its best to keep 2 functions.

Reply:
I'm hesitant to agree as interfaces should remain simple but we can capture it for later and figure it out then.